app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -191,6 +190,17 @@ $large-input-size: 50px;
   }
 }
 
+.gem-c-search--homepage {
+  .gem-c-search__submit {
+    background-color: #d2e2f1;
```

Comment:
I see the point in storing this colour somewhere but I personally I don't think brand-colours is the right place since that's a mixin for generating classes from the `$govuk-colours-organisations` defined by govuk-frontend (with some manual overrides). Maybe it's an issue I could raise is that there could be a place to put shared colours throughout govuk_publishing_components?

Reply:
Raising an issue sounds good and we can then look at it separately, it is a bit bigger than this particular ticket 👍